workflows/common/structs.wdl

```diff
@@ -25,11 +27,21 @@ struct SmrtcellInfo {
 
 struct SampleInfo {
   String name
-  Array[String?] parents
+  Boolean? affected
+  Array[String] parents
   Array[SmrtcellInfo] smrtcells
 }
 
-struct CohortInfo {
-  Array[SampleInfo] affected_persons
-  Array[SampleInfo] unaffected_persons
-}
\ No newline at end of file
+struct PacBioInfo {
+  String name
+  String path
+  Array[String] movie
+}
+
+struct PacBioSampInfo {
+  String name
+  Array[String] parents
```

Comment:
@cbi-star for Singletons will you still need to provide any empty array for the parents?

Reply:
yes